evap/staff/views.py

```diff
@@ -231,6 +231,9 @@ class Stats:
     degree_stats_with_total = cast(dict[Degree | str, Stats], degree_stats)
     degree_stats_with_total["total"] = total_stats
 
+    for evaluation in evaluations:
+        evaluation.has_exam = evaluation.course.evaluations.filter(name_de="Klausur", name_en="Exam").exists()
```

Comment:
I wonder if this is the right place to keep the knowledge of "what is an exam evaluation". Maybe `Evaluation` should have `exam_evaluation` helper that returns the queryset of all / the one exam evaluation(s)?

Reply:
Sounds good. Optimally, we'd also have the same knowledge about how to _make_ an exam evaluation in the same place, but that might not be so easy